components/menus/hub.ts

```diff
@@ -193,15 +193,16 @@ export function getHubData(gameVersion: GameVersion, userId: string) {
     }
 
     for (const child in locations.children) {
+        const location = locations.children[child]
+
         if (
             child === "LOCATION_ICA_FACILITY_ARRIVAL" ||
```

Comment:
is this still the case?

Reply:
Hiding arrival? Yes. Otherwise, I'm not sure what you're asking.

Reply:
Yes, IsHidden was added specifically for the Evergreen locations. Hence the comment about the HideProgression key (which is set to true for Arrival).